recipe/meta.yaml

```diff
@@ -1,38 +1,46 @@
 {% set name = "openmmforcefields" %}
-{% set version = "0.11.1" %}
+{% set version = "0.11.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/openmm/openmmforcefields/archive/{{ version }}.tar.gz
-  sha256: 22a201ee31c2e3f35dda9937aa87ef279aa96f4d8fd73f63c656774a202b4aa0
+  sha256: 89888ec81f539323fcf8fba8403adfa40c483f16b978b5861f65afaabe3b2615
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8
     - pip
   run:
-    - python >=3.6
+    - python >=3.8
     - openmm >=7.6.0
     - pyyaml
     - ambertools >=20.0
     - lxml
     - networkx
     - tinydb
-    - openff-toolkit >=0.9.0,<0.11.0
-    - openff-forcefields >=1.2.0
     - validators
 
+  run_constrained:
+    - openff-toolkit >=0.9.0
+    - openforcefield =9999999999999
```

Comment:
This basically means "do not install this package" - IIRC we moved `openforcefield` -> `openff-toolkit around January 2021 and bumped the version to 0.9.0